data/human/ships.txt

```diff
@@ -3474,6 +3474,68 @@ ship "Surveillance Drone"
 
 
 
+ship "Valkyrie"
+	sprite "ship/pirate valkyrie"
+	thumbnail "thumbnail/pirate valkyrie"
+	attributes
+		category "Medium Warship"
+		"cost" 3900000
+		"shields" 4800
+		"hull" 2800
+		"required crew" 9
+		"bunks" 26
+		"mass" 157
+		"drag" 4.9
+		"heat dissipation" .63
+		"fuel capacity" 500
+		"cargo space" 41
+		"outfit space" 400
+		"weapon capacity" 170
+		"engine capacity" 76
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Quad Blaster Turret" 2
+		"Particle Cannon" 2
+		"Torpedo Launcher" 2
+		"Torpedo" 60
+		
+		"NT-200 Nucleovoltaic"
+		"KP-6 Photovoltaic Panel" 3
+		"LP072a Battery Pack"
+		"D67-TM Shield Generator"
+		"Large Radar Jammer"
+		"Laser Rifle" 18
+		
+		"Greyhound Plasma Thruster"
+		"Greyhound Plasma Steering"
+		"Hyperdrive"
+		
+	engine 15 97
+	engine -15 97
+	gun -12 -78 "Particle Cannon"
+	gun 12 -78 "Particle Cannon"
+	gun -64 -19 "Torpedo Launcher"
+	gun 64 -19 "Torpedo Launcher"
+	turret -31 15.5
+	turret 31 15.5
+	bay "Fighter" -43.5 2
+	bay "Fighter" 43.5 2
```

Comment:
```suggestion
	gun -64 -19 "Torpedo Launcher"
	gun 64 -19 "Torpedo Launcher"
	turret -31 15.5
	turret 31 15.5
	bay "Fighter" -43.5 2
	bay "Fighter" 43.5 2
	bay "Fighter" -43.5 2
	bay "Fighter" 43.5 2
```

I am disappointed to see yet another carrier that emphasizes being a front-line combatant and downplays being a carrier. Why not actually give pirates something that could actually be a proper carrier?

There's few-to-no good carriers available outside of campaigns. This could be a good option for people... If it actually emphasized the carrier part. And to be honest, carriers very much suit pirates. What better to facilitate fast looting and pillaging to unleash a swarm of fighters across a battlespace that each can pick up a dozen tons of cargo? Much faster than having to lug their mothership to the four corners of the system. And faster looting means faster escape. Not to mention, depending on their strategies, they could opt for combat focused fighters or more cargo in the form of boxwings.

Reply:
We aren't going to add more fighter bays to a Carrier that isn't represented by the model. This was already shot down before when I attempted to buff the Aerie, as if we do want to add fighters to things, it seems the desire is to do so uniformly across the ships' variants and carriers in general. Making an exception like this doesn't seem proper at all.

Reply:
We're content creators, we make models to serve what the game needs, not twist the game to match the models we have. 

As for particle cannons... Who said I dislike them? I think they're great weapons. Maybe a bit too great, honestly. I love using them, and I can quite easily see why everyone else would want to use them. My only problem is that they're so common. By the end of the FW campaign it feels like everyone of any significance is using particle cannons. Navy? Yep. Syndicate? Yep. Extremists? Yep. Now we'll have pirates using them too? Sure, why not.

The point is balance. We have a whole range of weapons for a reason, and a large part of that reason is to provide a variety of challenges to players and to push the player to use different strategies. If we have a gun that is so good that it is both the only choice for long range support ships and for brawlers (who are the ones who will be in the center of deathballs)... Then we have a problem, because that basically means this gun is just the best at everything.